packages/legend-studio/src/components/editor/edit-panel/mapping-editor/InstanceSetImplementationEditor.tsx

```diff
@@ -346,6 +350,54 @@ export const InstanceSetImplementationSourceExplorer = observer(
   },
 );
 
+const MappingFilterEditor = observer(
+  ({
+    editorStore,
+    instanceSetImplementationState,
+    filterState,
+    isReadOnly,
+  }: {
+    editorStore: EditorStore;
+    instanceSetImplementationState: PureInstanceSetImplementationState;
+    filterState: PureInstanceSetImplementationFilterState;
+    isReadOnly: boolean;
+  }) => {
+    useEffect(() => {
```

Comment:
The call for the filterState's convertLambdaObjectToGrammarString is here but this doesn't trigger the reaction for calculating the diff as it does in propertyMappingState's transform objects.
@akphi @MauricioUyaguari